rest/model/task_test.go

```diff
@@ -123,6 +129,12 @@ func TestTaskBuildFromService(t *testing.T) {
 						AgentLogLink:  utility.ToStringPtr("url/task_log_raw//0?type=E"),
 						EventLogLink:  utility.ToStringPtr("url/event_log/task/"),
 					},
+					ParsleyLogs: LogLinks{
+						AllLogLink:    utility.ToStringPtr("parsley/evergreen//0/all"),
+						TaskLogLink:   utility.ToStringPtr("parsley/evergreen//0/task"),
+						SystemLogLink: utility.ToStringPtr("parsley/evergreen//0/system"),
+						AgentLogLink:  utility.ToStringPtr("parsley/evergreen//0/agent"),
```

Comment:
Does this test make sense? In what situation would there not be a taskid to populate this?

Reply:
i have no idea haha but it's an existing test; I think changing it is out of scope and it's not really harming anything